crypto/fipsmodule/curve25519/curve25519.c

```diff
@@ -103,6 +103,18 @@ void ED25519_keypair_from_seed(uint8_t out_public_key[ED25519_PUBLIC_KEY_LEN],
     ED25519_PUBLIC_KEY_LEN);
 }
 
+static void ed25519_keypair_pct(uint8_t public_key[ED25519_PUBLIC_KEY_LEN],
+  uint8_t private_key[ED25519_PRIVATE_KEY_LEN]) {
+#if defined(AWSLC_FIPS)
+  uint8_t msg[16] = {16};
+  uint8_t out_sig[ED25519_SIGNATURE_LEN];
+  if (ED25519_sign_no_self_test(out_sig, msg, 16, private_key) != 1 ||
+      ED25519_verify_no_self_test(msg, 16, out_sig, public_key) != 1) {
+    abort();
```

Comment:
we call `BORINGSSL_FIPS_abort()` for other PCTs.

Reply:
ahh doh. I forgot to switch it back to that after debugging,,,